test/commands/prodly/nut/checkIn.nut.js

```diff
@@ -0,0 +1,99 @@
+import { execCmd } from '@salesforce/cli-plugins-testkit';
+import { expect } from 'chai';
+describe('CLI CheckIn Commands Automation', () => {
+  // 1. Check-in With all the correct Flag
+  it('Check-In commands with all the flags using Dataset flag with Name', () => {
+    const rv = execCmd(
+      "prodly:checkin -i 'c31b1cde-7520-4f93-8b54-7347415764f3' -b 'main' -t 'Automation Dataset Account' -c 'Automation Test CLI checkin with all the flags' --json"
```

Comment:
Move this value into an env variable. We will then setup the variable in github action too.